solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/Theme/Constants.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Constants.swift
+//  SpaceApp
+//
+//  Created by Gustavo Soares on 10/03/22.
+//
+
+import SwiftUI
+
+enum SpaceApp {
```

Comment:
**[SUGGESTION]**
This namespace is to generic, consider using something more fit for localization domain like:
```suggestion
enum L10n {
```

Reply:
Great! To be honest with you, I had never followed it that way, i followed just as it are with constans and mapping strings there, but I liked it, thanks 🚀